src/transport_picoquic.cpp

```diff
@@ -786,13 +805,32 @@ PicoQuicTransport::close([[maybe_unused]] const TransportContextId& context_id)
 }
 
 void
-PicoQuicTransport::send_next_datagram(StreamContext* stream_cnx, uint8_t* bytes_ctx, size_t max_len)
-{
-    static auto last_time = std::chrono::steady_clock::now();
+PicoQuicTransport::check_callback_delta(StreamContext* stream_cnx, bool tx) {
     auto now_time = std::chrono::steady_clock::now();
-    auto delta_ms = std::chrono::duration_cast<std::chrono::milliseconds>(now_time - last_time).count();
 
+    if (tx) {
```

Comment:
Maybe 
```cpp
if (!tx) return;
```
?

Reply:
I can put that in, but the thought is that we'll have an else{} later.  We can add that later.